src/main/webapp/app/modules/administration/user-management/user-management-update.tsx

```diff
@@ -45,16 +51,23 @@ export const UserManagementUpdate = (props: IUserManagementUpdateProps) => {
   };
 
   const isInvalid = false;
-  const { user, loading, updating } = props;
+  const { user, loading, updating, featureFlags } = props;
 
-  const defaultValues = () => {
+  const defaultValues = useMemo(() => {
     return isNew
       ? {}
       : {
           ...user,
           authorities: user?.authorities?.map(auth => ({ label: auth, value: auth })),
+          featureFlags: user?.featureFlags?.map(f => ({ label: f.name, value: f })),
         };
-  };
+  }, [user]);
+
+  useEffect(() => {
+    setIsDev(props.user?.authorities?.includes(USER_AUTHORITY.ROLE_DEV));
+  }, [user]);
```

Comment:
Let's destructure the `props` on line 16, so you don't need to call `props.[variable]` everytime. Can get rid of line 54 after making that change.